en/docs/install-and-setup/setup/api-controller/encrypting-secrets-with-ctl.md

```diff
@@ -0,0 +1,125 @@
+# Encrypting Secrets with CTL
+
+WSO2 API Controller, **apictl** allows you to encrypt a plain-text secret. You can use this feature to export secrets as environment variables, system properties, Docker secrets, or Kubernetes secrets. For more information on using dynamic secrets refer [Dynamic secrets]().
```

Comment:
Link missing for page `Dynamic secrets`

Reply:
The page for Dynamic secrets hasn't moved yet from EI docs to APIM docs. Once it's completed, we need to update this.